pkg/config/remote/client.go

```diff
@@ -87,7 +88,7 @@ type agentGRPCConfigFetcher struct {
 // NewAgentGRPCConfigFetcher returns a gRPC config fetcher using the secure agent client
 //
 //nolint:revive // TODO(RC) Fix revive linter
```

Comment:
Question❓ Did this get fixed with your changes? i.e. can this comment be deleted?

Reply:
I may have missed one. I'll double check.

Reply:
I fixed it but forgot to remove the comment, good catch! Done!